lib/components/tab.vue

```diff
@@ -7,8 +7,7 @@
                    :aria-hidden="localActive ? 'false' : 'true'"
                    :aria-expanded="localActive ? 'true' : 'false'"
                    :aria-lablelledby="controlledBy || null"
-                   v-if="localActive || !lazy"
-                   v-show="localActive || lazy"
+                   v-show="localActive"
```

Comment:
what if this was:

```html
v-if="localActive && lazy"
v-show="localActive"
```
Would that make a difference?

Some people may not want any child components of a tab rendered if not visible.